README.md

```diff
@@ -52,7 +52,9 @@ The Adobe Experience Platform Mobile SDK provides support for the following plat
 * iOS - Swift \(New! - see [Current SDK versions](resources/upgrading-to-aep/current-sdk-versions.md#android) for repos, download links\)
 * iOS - Objective C \(includes iOS, iPadOS, and tvOS\) \(see [Current SDK versions](resources/upgrading-to-aep/current-sdk-versions.md#ios) for download links\)
 * Cordova \(see [Current SDK versions](resources/upgrading-to-aep/current-sdk-versions.md#cordova) for download links\)
-* React Native \(see [Current SDK versions](resources/upgrading-to-aep/current-sdk-versions.md#react-native) for download links\)
+* React Native (ACP 2.x) \(see [Current SDK versions](resources/upgrading-to-aep/current-sdk-versions.md#react-native-(acp-2.x)) for download links\)
+* React Native (AEP 1.x) \(see [Current SDK versions](resources/upgrading-to-aep/current-sdk-versions.md#react-native-(aep-1.x)) for download links\)
```

Comment:
Let's keep the URLs simple `#react-native-acp` and `#react-native-aep`. I'm sure you can supply anchor text on Current SDK Versions page for the respective sections.

Reply:
Let's also highlight by using `New!` like we do for iOS Swift. Also, let's move this before acp react native.
```
React Native (AEP 1.x) \(New! - see [Current SDK versions](...)) for download links\)
```